package.json

```diff
@@ -34,7 +34,6 @@
     "mem": "^5.1.1",
     "mixin-deep": "^2.0.1",
     "set-value": "^3.0.1",
-    "tailwindcss": "^2.1.2",
```

Comment:
PostCSS appears to still be required since we webpacker was upgraded recently. I'd love to see if we can remove it all together in the near future!